TestFoundation/TestNSString.swift

```diff
@@ -227,6 +228,18 @@ class TestNSString : XCTestCase {
         let string = NSString(CString: bytes.map { Int8(bitPattern: $0) }, encoding: NSUTF8StringEncoding)
         XCTAssertNil(string)
     }
+    
+    func test_FromContentOfFile() {
+        let testFilePath = testBundle().pathForResource("NSStingTestData", ofType: "txt")
```

Comment:
@joshuatbrown ,Im sorry for this,it's my stupid mistake~now,I fixed it,but it cant build while I run my test now, so I merge the newest code to my fork branch,but also not work, the error is in the 246 line of NSHTTPCookie.swift, error message is 

       /swift-corelibs-foundation/Foundation/NSHTTPCookie.swift:246:13: 
       All stored properties of a class instance must be initialized before returning nil from an initializer